docs/site/Best-practices.md

```diff
@@ -4,7 +4,7 @@ title: 'Best practices with Loopback 4'
 keywords: LoopBack 4.0, LoopBack 4
 tags:
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Best-practices-with-Loopback-4.html
+permalink: /doc/en/lb4/Best-practices.html
```

Comment:
Can we include a `redirect_from` for the original page as shown here: https://github.com/jekyll/jekyll-redirect-from

Reply:
Done, LGTY @virkt25 now?